### ISSUE TITLE
Is it possible to pass extra props to component?

### ISSUE THREAD

**Issue by vbabenko** [hilongjw/vue-recyclerview#12]
Currently it looks like this:
`<RecyclerView
      :prerender="30"
      :fetch="MiFetch" 
      :item="MiItem" 
      :tombstone="MiTomstone"
    ></RecyclerView>`

and `RecyclerView` will pass only one props `data` to item.

Is it possible somehow to pass extra props like this
`<RecyclerView
      :prerender="30"
      :fetch="MiFetch" 
      :item="MiItem" 
      :tombstone="MiTomstone"
      :propOne="myProp1"
      :propTwo="myProp2"
    ></RecyclerView>`
Thanks

**Comment by hilongjw** [comment]
Hi @vbabenko , I think: ` propTwo= 'myProp2' is not a good way, the name of the props here can not be customized, can only be fixed to propTwo, propThree etc.

I think the way through options will be better

```javascript
options: {
  props: {
    user: user,
    other: otherData
    ...
  }
}
```

```html
<RecyclerView 
:prerender="30"
:remain="30"
 :fetch="MiFetch"
 :item="MiItem" 
:tombstone="MiTomstone"
 :options="options">
</RecyclerView>
```
item renderer
```javascript
vm = new this.Vue({
      el: el,
      render: h => h(this.itemRender, {
        props: {
          data: data
          ...options.props
        }
    })
 })
```

**Comment by vbabenko** [comment]
@hilongjw Thanks, it works

**Comment by hazzo** [comment]
Sorry @hilongjw but I think that this option of passing props its not available in the last npm build. Could be possible? Because my source file it's not the same as the one here in github.

Thanks again

**Comment by hazzo** [comment]
Heheh also `@inited` is missing. I think you are on version 0.3.4 and NPM staid on 0.3.3

**Comment by hilongjw** [comment]
@hazzo  I just published 0.3.5 on npm

**Comment by hazzo** [comment]
Thanks!
